### DISCUSSION TITLE
Turborepo Vercel access built files of another app

### DISCUSSION THREAD

**Discussion by lewis-wow** [vercel/turborepo#5949] [Help]
### Question

I have two apps in turborepo, graphql and web application, graphql is written in nexus that build me graphql schema on build command in graphql app. Web application uses graphql codegen to generate queries and mutations. But I cannot access built/generated files at graphql app from web app in Vercel deployment even if I have marked the "Include source files outside of the Root Directory in the Build Step" checkbox, so codegen does not work. Locally it works fine.

Codegen config:

```ts
const config: CodegenConfig = {
  schema: '../graphql/schema.graphql',
  generates: {
    'src/generated/graphql.ts': {
      documents: './src/graphql/*.ts',
      config: {
        maybeValue: 'T | null | undefined',
      },
      plugins: ['typescript', 'typescript-operations'],
    },
  },
};
```

**Comment by anthonyshew** [comment]
Could you share your `turbo.json`?
